src/query/execute_statement.c

```diff
@@ -17133,3 +17074,95 @@ do_send_plan_trace_to_session (PARSER_CONTEXT * parser)
       free_and_init (plan_str);
     }
 }
+
+static int
+do_insert_checks (PARSER_CONTEXT * parser, PT_NODE * statement, PT_NODE ** class_, PT_NODE ** update, PT_NODE * values)
+{
+  int error = NO_ERROR;
+  int upd_has_uniques = 0;
+  bool has_default_values_list = false;
+
+  /* Check if server allows an insert. */
```

Comment:
@lucianlazar2211 

Please add
```C
*update = NULL;
```

Our coding conventions require to initialize an output parameter to make it sure.